src/tests/test_notifications.py

```diff
@@ -17,7 +17,9 @@ def test_send_email_with_valid_email(self):
         recipients email addresses
         """
         self.assertTrue(send_email(
-            sender=current_app.config["SENDER_CREDS"],
+            sender=("Andela Societies Test"
```

Comment:
Perhaps this might be better placed in the TestingConfig class. Adding the SENDER_CREDS class attribute will mean it's available throughout all tests for whoever might need to use it.

Reply:
You are right. Reverting accordingly.

Reply:
Got it. 
```
When the logger gets an email, who will they respond to? There's nothing in the email to point them to who requested more information.```
Still waiting for a response on that though.

Reply:
@Andretalik